===================================================================
RCS file: /home/freetype/cvs/cvs/freetype/freetype2/src/type1/t1load.c,v
retrieving revision 1.88
retrieving revision 1.89
diff -u -r1.88 -r1.89
--- freetype2/src/type1/t1load.c	2003/12/12 15:38:39	1.88
+++ freetype2/src/type1/t1load.c	2003/12/22 11:35:36	1.89
@@ -1527,6 +1527,11 @@
         parser->root.cursor = cur2;
       }
 
+      /* look for `eexec' */
+      else if ( *cur == 'e' && cur + 5 < limit &&
+                ft_strncmp( (char*)cur, "eexec", 5 ) == 0 )
+        break;
+
       /* look for `closefile' which ends the eexec section */
       else if ( *cur == 'c' && cur + 9 < limit &&
                 ft_strncmp( (char*)cur, "closefile", 9 ) == 0 )
